backend/database/migrations/2023_01_25_033418_user_tag.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user_tag', function (Blueprint $table) {
```

Comment:
Please follow the naming standard for pivot table, name should be alphabetical order, this way we can avoid specifying the schema table in models.
```suggestion
        Schema::create('tag_user', function (Blueprint $table) {
```